src/bundler/bundle_v2.zig

```diff
@@ -7546,6 +7548,90 @@ pub const LinkerContext = struct {
         return hasher.digest();
     }
 
+    pub fn validateTLA(c: *LinkerContext, source_index: Index.Int) js_ast.TlaCheck {
+        var result_tla_check: *js_ast.TlaCheck = &c.parse_graph.ast.items(.tla_check).ptr[source_index];
+        const result = c.parse_graph.ast.get(source_index);
```

Comment:
Please don't get the entire AST for only a handful of necessary fields

Reply:
https://github.com/oven-sh/bun/pull/14218/commits/b6add9c5314fddbdecff3f925a6bd09fd6517e1c